common/datavalues/src/types/deserializations/date_time.rs

```diff
@@ -69,9 +69,17 @@ where
     }
 
     fn de_text_quoted(&mut self, reader: &mut CpBufferReader) -> Result<()> {
-        reader.must_ignore_byte(b'\'')?;
+        reader
+            .must_ignore_byte(b'\'')
+            .map_err_to_code(ErrorCode::NoneBtBadBytes, || {
+                "Invalid start of datetime string"
+            })?;
         let datetime = reader.read_datetime_text(&self.tz)?;
```

Comment:
This has an exception backtrace...

Reply:
I also use `NoneBtBadBytes` in `read_datetime_text` and `read_date_text`.